src/components/link/link.spec.js

```diff
@@ -44,7 +44,7 @@ describe("Link", () => {
           paddingLeft: "24px",
           paddingRight: "24px",
           lineHeight: "36px",
```

Comment:
```suggestion
```
I think you can remove this style rule since the line height is included in `typographyLinkTextL`

Reply:
Good shout. I think I tried to remove it and it made `Menu` even worse but hadn't tried since I added the line height to `menu item`